sdk/src/network-client.ts

```diff
@@ -390,9 +392,9 @@ class AleoNetworkClient {
    * @example
    * const latestHeight = networkClient.getLatestHeight();
    */
-  async getLatestHeight(): Promise<number | Error> {
+  async getLatestHeight(): Promise<bigint | Error> {
```

Comment:
Block height should likely stay `number` in return type because block height is a u32 (and thus within the range of `number`) and several methods take block height as a `number` type which would lead to a frustrating explicit cast for end users. 

Even though with `ParseJson` we'd get it back as a big int, it'll always be a safe conversion to `number`, so we should probably do that directly in this method. 

Otherwise treating all numbers in the block JSON as BigInt is a safe approach as there are a ton of numbers represented as `u64` (and even the stray `u128` here and there) in rust.

Reply:
Yeah that's reasonable, I made that change.